.github/workflows/develop.yml

```
@@ -163,16 +163,32 @@ jobs:
+          - package-manager: bun
+            template: aws-lambda
+          - package-manager: bun
+            template: aws-lambda-websocket
+          - package-manager: bun
+            template: cloudflare-workers
+          - package-manager: bun
+            template: cloudflare-websocket
+          - package-manager: bun
+            template: nextjs
+          - package-manager: bun
+            template: nextjs-full
```

actionlint (1.7.12)

[error] value "nextjs-full" in "exclude" does not match in matrix "template" combinations. possible values are "express", "express-middleware", "fastify", "fastify-plugin", "uws", "ws", "aws-lambda", "aws-lambda-websocket", "cloudflare-workers", "cloudflare-websocket", "nextjs", "mcp-server", "pg-boss", "bullmq", "workflows-bullmq", "workflows-pg-boss", "remote-rpc-pg", "remote-rpc-redis", "ai-postgres", "bun"

(matrix)